src/main/java/seedu/address/logic/parser/AbsentAttendanceCommandParser.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.AbsentAttendanceCommand;
+import seedu.address.logic.commands.AbsentAttendanceCommand.PetAttendanceDescriptor;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new AbsentAttendanceCommand object.
+ */
+public class AbsentAttendanceCommandParser implements Parser<AbsentAttendanceCommand> {
```

Comment:
See previous comment at `PresentAttendanceCommand.java`! If have an interface for AttendanceCommand, maybe don't need 2 parsers for attendance.